docs/collector-components.md

```diff
@@ -0,0 +1,131 @@
+    one --> two --> three --> four --> five --> six
+
+    style one fill: #e6f9f7,stroke:#333,stroke-width:1px
+    style two fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style three fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style four fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style five fill: ##f8e9e9,stroke:#333,stroke-width:1px
+    style six fill: #e6f9f7,stroke:#333,stroke-width:1px
+```
+
+Refer to the following tables for more information on the components supported by the Elastic Distribution for the OpenTelemetry Collector.
+Follow the links for OpenTelemetry documentation with more configuration details for each component.
+To set up the collector, go to the [guided onboarding](guided-onboarding.md) or [manual configuration](manual-configuration.md) to get started.
+
+## Receivers
+
+| Component  | Description |
+|---|---|
+| [`filelogreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/filelogreceiver/v0.105.0/receiver/filelogreceiver/README.md) | Collects logs from files on the local filesystem, supporting various formats and log rotation strategies. |
+| [`hostmetricsreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/hostmetricsreceiver/v0.105.0/receiver/hostmetricsreceiver/README.md) | Collects metrics from the host machine, such as CPU, memory, disk, and network usage. |
+| [`httpcheckreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/httpcheckreceiver/v0.105.0/receiver/httpcheckreceiver/README.md) | Performs HTTP checks to monitor the availability and response time of web services. |
+| [`k8sclusterreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/k8sclusterreceiver/v0.105.0/receiver/k8sclusterreceiver/README.md) | Gathers metrics and metadata from a Kubernetes cluster. |
+| [`k8sobjectsreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/k8sobjectsreceiver/v0.105.0/receiver/k8sobjectsreceiver/README.md) | Monitors changes to Kubernetes objects, and collects related metrics. |
+| [`kubeletstatsreceiver`](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/receiver/kubeletstatsreceiver/v0.105.0/receiver/kubeletstatsreceiver/README.md) | Collects metrics from the Kubelet, including node and pod-level resource usage. |
+| [`otlpreceiver`](https://github.com/open-telemetry/opentelemetry-collector/blob/receiver/otlpreceiver/v0.105.0/receiver/otlpreceiver/README.md) | Receives metrics, traces, and logs in OpenTelemetry Protocol (OTLP) format. |
+
+## Processors
+
+| Component  | Description |
+|---|---|
+| [`elasticinframetricsprocessor`](https://github.com/elastic/opentelemetry-collector-components/blob/processor/elasticinframetricsprocessor/v0.7.1/processor/elasticinframetricsprocessor/README.md)  | Processes infrastructure metrics to enhance and convert them for Elasticsearch. |

```

Comment:
As of today this is not an upstream component. @AlexanderWert lets chat about this.